.github/CHANGELOG.md

```diff
@@ -21,6 +24,14 @@
 
 ### Improvements
 
+* Split matrix operations, refactor dispatch mechanisms, and add a benchmark suits.
```

Comment:
benchmark suite

Reply:
Ah! It's fixed in the release notes and will update CHANGELOG consequently. Thanks!